userguide/virtualmachines.rst

```diff
@@ -140,7 +167,7 @@ Virtual machine configuration options are described in
    | Screen # | Setting            | Value          | Description                                                                                   |
    |          |                    |                |                                                                                               |
    +==========+====================+================+===============================================================================================+
-   | 1        | Virtual Machine    | drop-down menu | Choose to create either a standard VM or a Docker Host.                                       |
+   | 1        | Virtual Machine    | drop-down menu | Choose to create either a standard VM or a :ref:`Docker Host <Docker Hosts>`.                 |
    |          | (VM) Wizard type   |                |                                                                                               |
```

Comment:
Not sure what this (VM) notation implies.

Reply:
It is an exact duplicate of the field name from the UI